### PR TITLE
adding code owners file to repo

### PR DESCRIPTION
**Pull request by LiquidPlummer**

### CODE REVIEW

**Comment by LiquidPlummer** [comment]
Just setting up branch protection and code owners. When we resume training after QC we'll get these changes reviewed and merged.

**Review by wallclockbuilder — DISMISSED** [review]
Write moar codes. 
--evil manager said

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*			@LiquidPlummer
+/server/	@LiquidPlummer
+/client/	@LiquidPlummer
```
Comment (Warmedpie) [inline .github/CODEOWNERS:3]: Why would you write this line??

**Comment by chadfitz** [comment]
I think only reviewers can review

**Review by Warmedpie — APPROVED** [review]
O(n^2) go for O(logn)

**Review by chadfitz — APPROVED** [review]
nice!

File: .github/CODEOWNERS (hunk 1 above)
Comment (wallclockbuilder) [reply .github/CODEOWNERS:3]: I am blocking everything

**Review by wallclockbuilder — APPROVED** [review]

**Comment by LiquidPlummer** [comment]
???

**Review by chadfitz — COMMENTED** [review]
might want more code owners

**Review by chadfitz — COMMENTED** [review]
okay

**Pull request merged**